src/PKSim.Core/Services/IndividualMoleculeFactory.cs

```diff
@@ -71,69 +78,48 @@ protected bool HasAgeParameter(ISimulationSubject simulationSubject)
          return simulationSubject?.Individual?.AgeParameter != null;
       }
 
-      protected ParameterValueMetaData RelExpParam(string paramName, double defaultValue = 0) => new ParameterValueMetaData
+      protected ParameterValueMetaData RelExpParam(string paramName, double defaultValue = 0)
       {
-         ParameterName = paramName,
-         Dimension = DIMENSIONLESS,
-         DefaultValue = defaultValue,
-         GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
-         BuildingBlockType = PKSimBuildingBlockType.Individual,
-         IsDefault = true,
-         MinValue = 0,
-         MinIsAllowed = true
-      };
-
-      protected ParameterRateMetaData FractionParam(string paramName, string rate,
-         bool editable = true, bool visible = true) =>
-         new ParameterRateMetaData
-         {
-            ParameterName = paramName,
-            Rate = rate,
-            CalculationMethod = CoreConstants.CalculationMethod.EXPRESSION_PARAMETERS,
-            BuildingBlockType = PKSimBuildingBlockType.Individual,
-            CanBeVaried = true,
-            CanBeVariedInPopulation = false,
-            ReadOnly = !editable,
-            Visible = visible,
-            Dimension = CoreConstants.Dimension.Fraction,
-            GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
-            IsDefault = true,
-            MinValue = 0,
-            MaxValue = 1,
-            MinIsAllowed = true,
-            MaxIsAllowed = true,
-         };
-
-      protected ParameterRateMetaData InitialConcentrationParam(string rate) =>
-         new ParameterRateMetaData
-         {
-            ParameterName = INITIAL_CONCENTRATION,
-            Rate = rate,
-            CalculationMethod = CoreConstants.CalculationMethod.EXPRESSION_PARAMETERS,
-            BuildingBlockType = PKSimBuildingBlockType.Individual,
-            CanBeVaried = true,
-            CanBeVariedInPopulation = false,
-            Dimension = MOLAR_CONCENTRATION,
-            GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
-            MinValue = 0,
-            MinIsAllowed = true
-         };
+         //In the db, we define those parameters as rate for now but we need to create a value meta data here so we update
+         var parameterMetaData = _parameterRateRepository.ParameterMetaDataFor(_containerPath, paramName);
+         var parameterValueMetaData = new ParameterValueMetaData {DefaultValue = defaultValue};
+         parameterValueMetaData.UpdatePropertiesFrom(parameterMetaData);
+         return parameterValueMetaData;
```

Comment:
I don't quite get this. Why do we need **Value** Metadata and not **Rate** metadata here?

Reply:
this is a very good question. I am just trying for now to be compatible with the old way,
In short the default value is always 0 except for our undefined enzymes where it's 1
At the moment, the value in the DB is NaN, I could set it to 0. I guess and go from there. 
Is RelativeExpression only define in one compartment ? Or does it depend on the organ?

Reply:
> Is RelativeExpression only define in one compartment ? Or does it depend on the organ?

We should only have **one** `RelativeExpression` per Organ. As we cannot have molecule properties on the organ level, I would suggest to put it into intracellular space.

Reply:
How is it NOW? Is it always in the Intracellular? Or do we have it sometimes in Lumen etc..
If we do, we have a problem

Reply:
You are right... It is always in `Intracellular` but also in lumen segments. Which is a questionable concept anyways...

Reply:
Agreed. This is because we cannot create molecule parameter at the organ level. It comes back to the same issue. 
In the meantime, since we do not want to change this now, what does this mean for what we are trying to achieve?
I cannot create a condition for RElativeExpression that would work as we except (e.g. Intracellular OR Lumen) since we do not have OR. If I leave the RelativeExpression default value to NaN, then we still have the problem that we cannot extend. It's easier to do probably than adding formulas left and right but this is all not ideal